### ISSUE TITLE
batch normalisation using gradient accumulation or multi-gpu

### ISSUE THREAD

**Issue by hengck23** [KaimingHe/deep-residual-networks#20]
During the initial training , how is the batch normalisation (bn) mean and variance computed? The batch size of 256 is large and has to be split into multiple sub-batches. In that case, each sub-batch mean and variance will be different? I am guessing the below, please let me know if I was wrong:
1. conduct training using sub-batch mean and variance, even though they are different
2. compute mean and variance using very large training samples
3. freeze the update for the convolutional weights. update bn parameters (scale and bias) using fixed mean and variance above
